### DISCUSSION TITLE
The ability to scroll the timeline (editor) at a fixed speed, not current bpm

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13886] [Ideas] [answered]
Hope this has not been suggested previously. If you need to move to a specific tick, you can hold down the shift key.

The fixed speed can be adjusted as a percentage of the track length (1%, 5%, etc.)

**Comment by peppy** [comment, marked as answer]
Is there an issue with using the seek bar at the bottom of the editor screen for this purpose?

**Reply by Moltanicaa** [reply]
This
![_Qbpsdu2peY.jpg](https://user-images.githubusercontent.com/59089206/125628205-e6d25e33-9712-4371-97ee-3fb0047cd2fb.jpg)

**Reply by peppy** [reply]
Let me try asking once more: Is there an issue with seeking using this bar at the bottom of the screen?

![20210714 224320 (dotnet)](https://user-images.githubusercontent.com/191335/125632480-90837917-a984-495f-9d6e-cc754ba8a683.gif)

The most common movement as a mapper is moving between beats, which is why the current seek logic is in place.

**Reply by Moltanicaa** [reply]
yes, indeed, completely out of my head
